util/build_docs.py

```diff
@@ -240,6 +254,12 @@ def main():
     os.environ["PATH"] += os.pathsep + str(hugo_localinstall_dir)
 
     try:
+        if not is_hugo_extended():
+            logging.info(
+                "Hugo extended (with SASS support) is required, but only non-extended version found."
```

Comment:
Add a space after the period.  This will print as:

```
[...]version found.Installing local [...]
```

Reply:
@silvestrst Sorry I missed this. Since this PR is already merged, can you do a follow-up PR?